sample/HttpTrigger-TypeScript/index.ts

```diff
@@ -0,0 +1,29 @@
+﻿export function index(context: any, req: any) {
```

Comment:
@christopheranderson is working on some of the type definitions, but the goal is to have at least the basic typings defined (e.g. context, request, response, etc.)